web/Gruntfile.js

```diff
@@ -9,6 +9,10 @@
 
 module.exports = function (grunt) {
 
+  // Add new languages here
+  // Do not put en or exclaim in this list
+  var i18nForeignLanguages = ['app/i18n/ar.json'];
```

Comment:
I don't know if written Arabic is more standardized than spoken Arabic, but my understanding is that there are wide regional variations in spoken Arabic, so this might need to be `ar-sa.json`. Maybe something the WB can answer?